boards/rcar_salvator_xs_m3.conf

```diff
@@ -13,3 +13,22 @@ CONFIG_PARTIAL_DEVICE_TREE_SIZE=70000
 
 CONFIG_AOS_NODE_ID="gen3-dom0"
 CONFIG_AOS_NODE_TYPE="zephyr-gen3-dom0"
+
+CONFIG_MINIMAL_LIBC_MALLOC_ARENA_SIZE=8192
+CONFIG_FS_LITTLEFS_FC_HEAP_SIZE=8192
+CONFIG_SHELL_STACK_SIZE=8192
+
+CONFIG_DISK_ACCESS=y
+CONFIG_DISK_DRIVERS=y
+CONFIG_DISK_DRIVER_MMC=y
+
+CONFIG_MMC_RCAR=y
+CONFIG_MMC_RCA=1
```

Comment:
(note) it would be nice to reduce `CONFIG_SD_DATA_TIMEOUT` if you don't have really long files, like less than 40MB always, you can reduce the timeout to 1 second

definitely, it is needed to rework how zephyr works with timeouts and we need to take into account speed of read/write ops for one block